displays/cavity_display/backend/fault.py

```diff
@@ -17,18 +17,19 @@
 
 @dataclasses.dataclass
 class FaultCounter:
-    fault_count: int = 0
+    alarm_count: int = 0
     ok_count: int = 0
     invalid_count: int = 0
+    warning_count: int = 0
 
     @property
     def sum_fault_count(self) -> int:
-        return self.fault_count + self.invalid_count
+        return self.alarm_count + self.invalid_count
```

Comment:
I don't believe we're using this anywhere at the moment, but I guess in the future if we do use it, we might want to include `self.warning_count` to this sum?

Reply:
I think I had the vague idea that warnings aren't as much of a problem as alarms and invalids but honestly who knows 😂

Reply:
Ah okay, that makes sense! I guess can revisit later depending on the use case whenever the function does get used:D